java/bigtable-proxy/src/test/java/com/google/cloud/bigtable/examples/proxy/commands/ServeTest.java

```diff
@@ -107,7 +115,7 @@ public void setUp() throws IOException {
             InProcessChannelBuilder.forName(TARGET_SERVER_NAME).usePlaintext().build());
 
     // Create the proxy
-    serve = createAndStartCommand(fakeServiceChannel);
+    serve = createAndStartCommand(fakeServiceChannel, fakeCredentials);
```

Comment:
Should we add a comment for some documentation about managing/instantiating credentials?

Reply:
done